datalab/datalab_session/analysis/raw_data.py

```diff
@@ -78,15 +76,9 @@ def calculate_bins(start, end, num_bins):
             bin_middles.append(previous_edge + int((edge-previous_edge) / 2.0))
         previous_edge = edge
 
-    # Using np.log10 on the histogram made some wild results, so just apply log10 to each value
-    hist = []
-    for h in histogram:
-        if h > 0:
-            hist.append(math.log10(h))
-        else:
-            hist.append(0)
+    hist = np.log10(np.maximum(histogram, 1))
```

Comment:
I'm really worried about doing this since I put the comment above on how this made wild results. So there is at least some inputs where this wont work and will produce strange results.